### ISSUE TITLE
focus the running app when moving from workspace to another

### ISSUE THREAD

**Issue by AbdoOmarEg** [widavies/WinJump#21]
first of all thank you for this amazing app, it helped me so much
Can it possibly focuses the app on the workspace, when moving from workspace to another workspace and I try to type I find myself typing on another file in previous workspace so I have to move to another workspace (alt + 2) and click on the app or alt + tab to be able to type on the current app in the current workspace and thank you again for this app, and sorry for my bad English.

**Comment by sbochoa** [comment]
> first of all thank you for this amazing app, it helped me so much Can it possibly focuses the app on the workspace, when moving from workspace to another workspace and I try to type I find myself typing on another file in previous workspace so I have to move to another workspace (alt + 2) and click on the app or alt + tab to be able to type on the current app in the current workspace and thank you again for this app, and sorry for my bad English.

I'm having the same issue, is there any workaround available to that? Apart from that, work like a charm, thanks.

**Comment by widavies** [comment]
@AbdoOmarEg @sbochoa are you both on the latest version? This may fix it:

![image](https://github.com/widavies/WinJump/assets/11671115/b2ac02de-a3d2-46d7-87c4-ca713560a624)

These focus issues are quite annoying, I know exactly what you're talking about. Windows has some weird quirk that causes this, often I've found the exact same WinJump version works on one of my computers but not the other. 

Linking [this issue](https://github.com/MScholtes/VirtualDesktop/issues/57#issuecomment-1518601787) in case we need to look at some more ideas for how to fix it.

**Comment by sbochoa** [comment]
Hi @widavies, you're right! I just installed the newest version, and it works as expected (at least for my usage). Thank you!

**Comment by AbdoOmarEg** [comment]
@widavies, the newest version unfocuses whenever I move from workspace to another but doesn't focus any window which means I have to alt + 2 then alt + tab to focus the new app, that's much better but not the intended solution that I was hoping for but if that's the only solution then I'm more than happy with this workaround, @sbochoa does this happen to you too?

**Comment by AbdoOmarEg** [comment]
@widavies sounds great, and take your time it still works great

**Comment by AbdoOmarEg** [comment]
It works perfectly now for some reason for about a week now and I don't know why It didn't work on the first day

**Comment by widavies** [comment]
@AbdoOmarEg I've had the same issue, sometimes it works, sometimes it doesn't. At some point I'll try to make it more reliable when I have a second.

**Comment by AbdoOmarEg** [comment]
@widavies yeah, it does, but it helped me with my productivity nonetheless.
